tests/crashes/132766.rs

```diff
@@ -0,0 +1,9 @@
+//@ known-bug: #132766
```

Comment:
Question:

> This looks like a modified version of tests/ui/inference/ice-ifer-var-leaked-out-of-rollback-122098.rs. It would be nice to indicate that you minimized it from an existing test in the future.

Is that test still present?